bulk-delete.php

```diff
@@ -441,6 +448,28 @@ public function add_delete_posts_settings_panel() {
 		wp_enqueue_script( 'postbox' );
 	}
 
+	/**
+	 * Ajax call back function for getting taxonomies to load select2 options.
+	 *
+	 * @since 5.7.0
+	 */
+	public function load_taxonomy_term(){
+		$return = array();
```

Comment:
Change the variable name `$return` to something that is more readable like `payload` or `response`.